primer/index.md

```diff
@@ -2024,561 +2025,194 @@ void read_from_file(){
     TFile in_file("my_rootfile.root");
 
     // Get the Histogram out
-    TH1F* h;
-    in_file.GetObject("my_histogram",h);
+    auto h = in_file.Get<TH1F>("my_histogram");
 
     // Draw it
     auto myCanvas = new TCanvas();
     h->DrawClone();
 }
 {% endhighlight %}
 
-## N-tuples in ROOT
-
-***Storing simple N-tuples***
+## The ROOT dataset
+ROOT offers its own data format with the TTree class. These are some of its advantages:
```

Comment:
Given that the ROOT file format is independent from TTree, I would suggest to rephrase this as:
```suggestion
ROOT can handle storage/retrieval of large datasets through the {% include ref class="TTree" %} class. These are some of its advantages:
```